self-tests.yml

```diff
@@ -603,6 +593,122 @@ tasks:
     tags: ["db", "race"]
     name: race-migrations
 
+  # lint tasks
+  - <<: *run-go-test-suite-with-mongodb
```

Comment:
Small nit, why do these need the database running? Otherwise, lgtm

Reply:
Oops, that was a mistake